TreeMaker/python/maker.py

```diff
@@ -100,6 +100,12 @@ def __init__(self,parameters):
         if self.dataset!=[] :    
             self.readFiles.extend( [self.dataset] )
 
+        if 'inputFiles' in self.parameters.params:

```

Comment:
how does this differ from the `dataset` option above?

Reply:
The `dataset` option had some issues last time I tried it. In any case I'll delete this now, and if I run into the problems I had with the `dataset` option I'll fix those then.